docs-2.0/5.configurations-and-logs/1.configurations/2.meta-config.md

```diff
@@ -0,0 +1,53 @@
+`stdout_log_file`               |`metad-stdout.log`              | Specifies the filename for the stdout log.
+`stderr_log_file`               | `metad-stderr.log`| Specifies the filename for the stderr log.
+`stderrthreshold`         | `2`     | Specifies the minimum level to copy the log messages to stderr. Available values are 0-3. `0`, `1`, `2`, and `3` are `INFO`, `WARNING`, `ERROR`, and `FATAL`. |
+
+## Networking configurations
+
+| Name                      | Default Value    | Descriptions                                                                                                                                                 |
+| ------------------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `meta_server_addrs`       | `127.0.0.1:9559` | Specifies the IP addresses for the Meta Services. Multiple addresses are permitted, separated by commas.                                                  |
+`local_ip`                      | `127.0.0.1`   | Specifies the local IP for the Meta Service.  |
+| `port`                    | `9559`            | Specifies RPC daemon listening port. The external port for Meta Service is `9559`. The internal port is `port+1`, namely `9560`. Nebula Graph uses the internal port for multi-replica interactions. |
+| `ws_ip`                   | `0.0.0.0`    | Specifies the IP address for the HTTP service.                                                                                                                                        |
+| `ws_http_port`            | `19559`            | Specifies the port for the HTTP service.                                                   |
+| `ws_h2_port`              | `19560`            | Specifies the port for the HTTP2 service.                     |
+|
+
+> **NOTE**: We recommend that you use the real IP address in your configuration because sometimes `127.0.0.1` can not be parsed correctly.
+
+## Storage configurations
+
+| Name        | Default Value                                 | Descriptions                               |
+| ----------- | --------------------------------------------- | ------------------------------------------ |
+| `data_path` | data/meta (i.e. /usr/local/nebula/data/meta/) | Directory for cluster metadata persistence |
+
+## Misc configurations
+
+| Name        | Default Value                                 | Descriptions                               |
+| ----------- | --------------------------------------------- | ------------------------------------------ |
+|`default_parts_num` | `100` | Specifies the default partition number when you create a new graph space. |
+`default_replica_factor` | `1` | Specifies the default replica factor number when you create a new graph space.
```

Comment:
>**CAUTION**: if you change this parameter `heartbeat_interval_secs` at a place, make sure to change all the configure files (including meta, graph, storage)  to the same value. Otherwise, the system behavior is undefined.